exporting/json/json.c

```diff
@@ -51,6 +51,44 @@ int init_json_instance(struct instance *instance)
     return 0;
 }
 
+/**
+ * Format host labels for JSON connector
+ *
+ * @param instance an instance data structure.
+ * @param host a data collecting host.
+ * @return Always returns 0.
+ */
+int format_host_labels_json_plaintext(struct instance *instance, RRDHOST *host)
+{
+    if (!instance->labels)
+        instance->labels = buffer_create(1024);
+
+    if (unlikely(!sending_labels_configured(instance)))
+        return 0;
+
+    buffer_strcat(instance->labels, "\"labels\":{");
+
+    int count = 0;
+    netdata_rwlock_rdlock(&host->labels_rwlock);
+    for (struct label *label = host->labels; label; label = label->next) {
+        if (!should_send_label(instance, label))
+            continue;
+
+        char value[CONFIG_MAX_VALUE * 2 + 1];
+        sanitize_json_string(value, label->value, CONFIG_MAX_VALUE);
```

Comment:
`value` variable has two times the size given as argument, should we not increase the argument value?

Reply:
No. The `value` variable should be twice as large as the `label->value` in case every symbol in the string is escaped.

Reply:
Ok, I got it.